crypto/dilithium/pqcrystals_dilithium_ref_common/packing.c

```diff
@@ -131,15 +140,15 @@ void unpack_sk(uint8_t rho[SEEDBYTES],
     tr[i] = sk[i];
   sk += TRBYTES;
 
-  for(i=0; i < L; ++i)
-    polyeta_unpack(&s1->vec[i], sk + i*POLYETA_PACKEDBYTES);
-  sk += L*POLYETA_PACKEDBYTES;
+  for(i=0; i < params->l; ++i)
+    polyeta_unpack(params, &s1->vec[i], sk + i * params->poly_eta_packed_bytes);
+  sk += params->l*params->poly_eta_packed_bytes;
 
-  for(i=0; i < K; ++i)
-    polyeta_unpack(&s2->vec[i], sk + i*POLYETA_PACKEDBYTES);
-  sk += K*POLYETA_PACKEDBYTES;
+  for(i=0; i < params->k; ++i)
+    polyeta_unpack(params, &s2->vec[i], sk + i * params->poly_eta_packed_bytes);
+  sk += params->k*params->poly_eta_packed_bytes;
```

Comment:
```suggestion
  sk += params->k * params->poly_eta_packed_bytes;
```

Reply:
Thanks, fixed in [ea859d1](https://github.com/aws/aws-lc/pull/1910/commits/ea859d1b4a527b801a531e415122f51475802817).